cpu/cc2538/include/cc2538_rf.h

```diff
@@ -41,6 +41,7 @@ extern "C" {
 #define CC2538_AUTOCRC_LEN          (2)
 #define CC2538_RF_FIFO_SIZE         (128)
 #define CC2538_PACKET_LENGTH_SIZE   (1)
+#define CC2538_LENGTH_BYTE_MASK     (0x7F)
```

Comment:
The CI wants this to be documented with a Doxygen comment.

Reply:
Added the comment and squashed the commits.